pkg/parquetconverter/converter.go

```diff
@@ -430,6 +440,13 @@ func (c *Converter) convertUser(ctx context.Context, logger log.Logger, ring rin
 
 		converterOpts := append(c.baseConverterOptions, convert.WithName(b.ULID.String()))
 
+		userConfiguredSortColumns := c.limits.ParquetConverterSortColumns(userID)
+		if len(userConfiguredSortColumns) > 0 {
```

Comment:
We can remove the if and always use `append(sortColumns, userConfiguredSortColumns...)` as sorting columns.